src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Container } from "@mui/material";
+import LoginForm from "../components/Forms/LoginForm";
+import { useUser } from "../contexts/UserContext";
+
+function LoginPage() {
+  const userContext = useUser()
+
+  return (
+    <Container maxWidth="md">
+      <h2>Logga in</h2>
+      <LoginForm />
+      <Box>
```

Comment:
Ska va ändra det till svenska? :D

Reply:
Vilket? :)

Reply:
All text: To log in as admin, To log in as user osv  :)

Reply:
Aha, det är bara placeholder text för oss som testar, ingenting som kommer att synas ut mot "kund" sedan. 
Så för egen del föredrar jag engelska i just development funktioner.